dolphin/drivers/helper.py

```diff
@@ -0,0 +1,36 @@
+# Copyright 2020 The SODA Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import abc
+from oslo_log import log
+
+from dolphin import exception
+from dolphin import db
+
+LOG = log.getLogger(__name__)
+
+
+def get_access_info(context, storage_id):
+    if not storage_id:
```

Comment:
Why need `if not storage_id` here
`storage_id` is not a default argument, it should have a given value instead of None

Reply:
Yes, it should have a given value, so None is invalid, and we cannot suppose the caller always give the value correctly. But it's not need to check here, cause the low level function will raise exception, already modified.